internal/xds/translator/testdata/out/xds-ir/http-route-zonal-routing.endpoints.yaml

```diff
@@ -0,0 +1,63 @@
+- clusterName: first-route-dest
+  endpoints:
+  - lbEndpoints:
+    - endpoint:
+        address:
+          socketAddress:
+            address: 1.1.1.1
+            portValue: 50001
+      loadBalancingWeight: 40
+    locality:
+      region: first-route-dest/backend/0
+      zone: us-west-2a
```

Comment:
Nit: can we add more endpoints with same/different zones in this `lbEndpoints` to make this test more meaningful?

Reply:
Thanks, that was my original intention but added this wrong 😅